charts/dapr/README.md

```diff
@@ -79,7 +79,7 @@ The Helm chart has the follow configuration options that can be supplied:
 | `global.tag`                              | Docker image version tag                                                | `1.2.1`                 |
 | `global.logAsJson`                        | Json log format for control plane services                              | `false`                 |
 | `global.imagePullPolicy`                  | Global Control plane service imagePullPolicy                            | `IfNotPresent`          |
-| `global.imagePullSecret`                  | Control plane service image pull secret for docker registry             | `""`                    |
+| `global.imagePullSecrets`                 | Control plane service image pull secret for docker registry             | `""`                    |
```

Comment:
Should the description also be plural?

"Control plane service image pulls secrets for docker registry"